src/mrinufft/trajectories/display.py

```diff
@@ -163,21 +164,21 @@ def _setup_3D_ticks(figsize, fig=None):
 
 
 def display_2D_trajectory(
-    trajectory,
-    figsize=5,
-    one_shot=False,
-    subfigure=None,
-    show_constraints=False,
-    gmax=DEFAULT_GMAX,
-    smax=DEFAULT_SMAX,
-    constraints_order=None,
-    **constraints_kwargs,
-):
+    trajectory: np.typing.NDArray,
```

Comment:
I think you can just `from numpy.typing import NDArray` and use it everywhere

Reply:
Done